build.sbt

```diff
@@ -48,6 +53,17 @@ lazy val dottyTensorflow = project
   )
   .dependsOn(scalaPyTensorFlowJVM)
 
+lazy val dottyTensorFlowExamples = project
+  .in(file("dotty-tensorflow-examples"))
+  .settings(
+    name := "dotty-tensorflow-examples",
+    scalaVersion := dottyVersion,
+    publish := {},
```

Comment:
This can be extracted to Build.scala as some `implicit class ProjectOps(...) { def disablePublish .. }`

Reply:
no need. new `sbt` function `publish / skip := true` skips all forms of publish 🎉